src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -467,7 +467,7 @@ def get_pvs(fields=PV_FIELDS, filters='', tags=None):
     :returns: list of class PVolume object representing pvs on the system
     """
     filters = make_filters_lvmcmd_ready(filters, tags)
-    args = ['pvs', '--no-heading', '--readonly', '--separator=";"', '-S',
+    args = ['pvs', '--noheadings', '--readonly', '--separator=";"', '-S',
```

Comment:
Do you know when this was changed? Do we need to care about older systems?

Reply:
lvm of lower version also support --noheadings, but not support --no-heading; higher version only support --noheadings, instead of --no-heading

Reply:
Do you know when `--noheadings` was introduced? Do we need to care about systemds that don't yet support `--noheadings`?

Reply:
All versions of lvm currently support --noheadings